CAFAna/Core/OscillatableSpectrum.cxx

```diff
@@ -107,14 +107,14 @@ namespace ana
 
     assert( rhs.fReferences.empty() ); // Copying with pending loads is unexpected
     assert( fReferences.empty() ); // Copying with pending loads is unexpected
-
+    
     return *this;
   }
 
   //----------------------------------------------------------------------
-  OscillatableSpectrum& OscillatableSpectrum::operator=(OscillatableSpectrum&& rhs)
+  OscillatableSpectrum &OscillatableSpectrum::operator=(OscillatableSpectrum &&rhs)
```

Comment:
This file is an example of random whitespace changes. The one here is a matter of style (which disagrees with the prevailing cafana core style), but there are others in this file that are just stray spaces for no reason etc.